CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+## Coding style
+
+* HOL source code
+  - Spaces, no tabs
+  - No unicode
+  - `snake_case` (e.g. `bir_number_of_mem_splits_def`)
+
+
+
+## Branch policy
+
+### tags
+
+tags should have as many **completed features** as possible:
+ - no cheat
```

Comment:
**See next comment about cheat in `master` before.**

Is that possible? If you tag on master, where development happens, then maybe there is some `cheat`?

Reply:
let's start with the good intentions and then we can gradually go back in case it is hard to keep them up. in case it is hard to follow up on this, we could tag instable modules or directories, where tests are not required to succeed (for example)

Reply:
:+1: That's what I had in mind: tag and mention tools status.

Reply:
let's build the support for this later and just mention it here for now. so we settle for a tagging of unstable directories which enables usage of cheat and failing tests in there

Reply:
and we build it as soon as we need it